pkg/sql/pg_catalog.go

```diff
@@ -2179,6 +2179,19 @@ CREATE TABLE pg_catalog.pg_proc (
 							argmodes = tree.DNull
 							variadicType = oidZero
 						}
+
+						var volatile byte
+						switch props.Volatility {
+						case tree.VolatilityImmutable:
+							volatile = 'i'
+						case tree.VolatilityStable:
+							volatile = 's'
+						case tree.VolatilityVolatile:
+							volatile = 'v'
+						default:
+							return errors.Newf("unexpected volatility %d on proname %s", props.Volatility, name)
```

Comment:
We should probably panic here. If it's something we don't recognize, that indicates a more major problem.